src/components/WinnerDeclaration.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
```

Comment:
I think you added an empty file `/src/components/WinnerDeclaration`. Make sure you delete it

Reply:
I renamed the file to include .js. This is a bug with renaming the file.